ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/behaviorTree/condition/ConditionNodeDefinition.java

```diff
@@ -19,7 +19,9 @@ public enum Type
    {
       COUNTER,
       LLM,
-      PROXIMITY;
+      PROXIMITY,
+      ALWAYS_FAIL,
+      ALWAYS_SUCCEED;
```

Comment:
Added always succeed and fail conditions, which are useful for testing and workarounds.